src/chapters/forms/form-submission.rst

```diff
@@ -0,0 +1,120 @@
+Form Submission
+===============
+
+.. index::
+   single: form; submission
+
+Forms collect data input by the user. As we learned in the previous
+chapter, communication on the web occurs via a series of HTTP requests and responses. A
+**form submission** is an HTTP request sent to the server containing the values
+in a form.
+
+Trigger Form Submission
+-----------------------
+
+A form submission is triggered by clicking a button inside the form. A submit button can be
+an ``input`` element with ``type=submit`` or a ``button`` element. Both button types are
+in the below example.
+
+.. sourcecode:: html
+   :linenos:
+
+   <form>
+      <label>Username <input type="text" name="username"></label>
+      <!-- clicking either of these will cause a form submission -->
+      <input type="submit"/>
+      <button>Submit</button>
+   </form>
+
+When a form is submitted, an HTTP request is sent to the location set in the ``action``
+attribute of the ``<form>`` tag.
+
+If the action attribute is not present or is empty, then the form will submit to the URL
+of the current page.
+
+.. admonition:: Try It!
+
+   Open `this form <https://form-default--launchcode.repl.co/>`_ in a browser.
+   Type values into the inputs, click the Submit button, and notice what happens to the
+   address bar.
+
+   .. sourcecode:: html
+      :linenos:
+
+      <html>
+         <head>
+            <title>Form Example</title>
+            <style>
+               body { padding: 25px;}
+            </style>
+         </head>
+         <body>
+            <form action="">
+               <label>Username <input type="text" name="username"></label>
+               <label>Team Name <input type="text" name="team"></label>
+               <button>Submit</button>
+            </form>
+         </body>
+      </html>
+
+   **Output**
+
+   .. figure:: figures/default-form.png
+      :alt: Browser screen shot showing form with two text inputs and a submit button. Both inputs have text values.
+
+   **Output After Submitted**
+
+   .. figure:: figures/default-form-submitted.png
+      :alt: Browser screen shot showing form after it has been submitted. The URL has queryString showing.
+
+   `Run it <https://repl.it/@launchcode/form-default>`_.
+
```

Comment:
I would take this out as it seems redundant, as it takes us to the replit page.  I would pick either open form in browser or replit link, and use it at line 37 and delete line 70